src/components/registration/registration-form/RegistrationForm.tsx

```diff
@@ -61,7 +62,13 @@ const RegistrationForm = ({ onSubmit, session }: RegistrationFormProps) => {
   return (
     <form
       data-testid={TestId.RegistrationForm}
-      onSubmit={(e) => onSubmit(e, family)}
+      onSubmit={(e) =>
+        onSubmit(e, {
+          family,
+          session: session.id,
+          preferred_class: null, // Change this in later task
```

Comment:
we can add `TODO:` in front so it's easier for us to find later, as well as a more descriptive message for what needs to happen!